docs/Database.md

```diff
@@ -5,6 +5,47 @@ The `CCT` and `MyGordon` databases exist in:
 - `admintrainsql.gordon.edu` - The development/testing database server
 - `adminprodsql.gordon.edu` - The live production database server.
 
+### Updating context tables (_new columns, tables, procedures..._)
+
+Entity Framework 6 Power Tools are design-time utilities used in Visual Studio when working with the code-first development approach. As you know, the code-first approach does not support visual designer for the Entity Framework model. These power tools allow you to generate designer, XML and DDL SQL for your entities. In order to utilize this extension:

```

Comment:
This says both more and less than Evan suggested. 

While there's obviously much more to say, there's enough here to help someone search for more online, and it's the instructions below that I think are most critical.